analyzers/its/expected/akka.net/S1694-Akka-netstandard2.0.json

```diff
@@ -6,24 +6,12 @@
       "Uri": "https://github.com/SonarSource/sonar-dotnet/blob/master/analyzers/its/Projects/akka.net/src/core/Akka/Dispatch/ExecutorService.cs#L53",
       "Location": "Line 53 Position 27-49"
     },
-    {
-      "Id": "S1694",
-      "Message": "Convert this \u0027abstract\u0027 class to an interface.",
-      "Uri": "https://github.com/SonarSource/sonar-dotnet/blob/master/analyzers/its/Projects/akka.net/src/core/Akka/Dispatch/Mailbox.cs#L554",
-      "Location": "Line 554 Position 27-38"
-    },
     {
       "Id": "S1694",
       "Message": "Convert this \u0027abstract\u0027 class to an interface.",
       "Uri": "https://github.com/SonarSource/sonar-dotnet/blob/master/analyzers/its/Projects/akka.net/src/core/Akka/IO/IO.cs#L15",
       "Location": "Line 15 Position 27-38"
     },
-    {
-      "Id": "S1694",
-      "Message": "Convert this \u0027abstract\u0027 class to an interface.",
-      "Uri": "https://github.com/SonarSource/sonar-dotnet/blob/master/analyzers/its/Projects/akka.net/src/core/Akka/Pattern/BackoffOptions.cs#L77",
```

Comment:
FP: in .NET Standard 2.0 interfaces cannot have [members with internal visibility](https://github.com/SonarSource/sonar-dotnet/blob/fd7a9bea40fa21361bd8e05fcbba8365f3648307/analyzers/its/Projects/akka.net/src/core/Akka/Pattern/BackoffOptions.cs#L122).